ScoutSuite/output/data/html/partials/gcp/services.cloudresourcemanager.bindings.html

```diff
@@ -1,12 +1,12 @@
 
```

Comment:
I think there was a bad merge here. The `.projects.` was added in one of Remi's refactoring.